src/settings/controllers/ThemeController.js

```diff
@@ -19,9 +19,17 @@ import SettingController from "./SettingController";
 import {DEFAULT_THEME, enumerateThemes} from "../../theme";
 
 export default class ThemeController extends SettingController {
+    static isLogin = false;
+
+    static setIsLogin(val) {
```

Comment:
any reason to not just call `isLogin` directly? We can always change it to a static setter if we need to, and the variable is public anyways.